### ISSUE TITLE
Set learning rate of `AdaBoost`

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stdlib#167]
### Is your feature request related to a problem?

It's not possible to set the learning rate of adaptive boosting.

### Desired solution

* Add a new parameter `learning_rate: float` to the initializer of `safeds.ml.classification.AdaBoost` and `safeds.ml.regression.AdaBoost`
* Raise a value error if `learning_rate` <= 0
* Pass it along to `learning_rate` of the wrapped `scikit-learn` model in the `fit` method
